src/funcTest/groovy/info/solidsoft/gradle/pitest/functional/AbstractPitestFunctionalSpec.groovy

```diff
@@ -36,8 +38,8 @@ abstract class AbstractPitestFunctionalSpec extends IntegrationSpec {
     }
 
     protected void writeHelloPitClass(String packageDotted = 'gradle.pitest.test.hello', File baseDir = getProjectDir()) {
-        def path = 'src/main/java/' + packageDotted.replace('.', '/') + '/HelloPit.java'
-        def javaFile = createFile(path, baseDir)
+        String path = 'src/main/java/' + packageDotted.replace('.', '/') + '/HelloPit.java'
+        File javaFile = createFile(path, baseDir)
```

Comment:
+1, that code had to be created long time ago :-).

Reply:
I'll leave as is, but FYI I'll have to disable NoDef rule so test methods can be declared with `def` instead of `void`, so these cases won't be picked up in future unless a different config is applied to test sources than production sources.

Reply:
No, I said "+1" that I like this change (and I was using `def` long time ago). Please change it.